### ISSUE TITLE
[Feature Request] Option to always snap to the bottom of updated preview

### ISSUE THREAD

**Issue by Zooce** [qjebbs/vscode-plantuml#128]
**TL;DR
I'd like to request an option to snap auto-updated preview windows to the bottom of the preview image.**

When viewing auto-updated preview, it always re-displays the preview at the top (i.e. the preview window is snapped to the top of the preview image). There's no problem with this for small diagrams, but for long sequence diagrams - the image grows downward. If i'm scrolled to the bottom of the preview image (where my most recent changes are), and add something new, then the preview is updated, and snapped back to the top of the new preview - so I again have to scroll back down to where I made my change to see it. With an option to snap the newly-generated preview to the bottom of the image, would get rid of this annoyance and (as a plus) increase users' productivity.

**Comment by qjebbs** [comment]
The snap feature could be a nice improvement, I will consider when I have time. 

But updated image will keep the last zoom and position, and won't go back to top, doesn't it?

![demo](https://user-images.githubusercontent.com/16953333/41751058-e8f55a16-75f1-11e8-90ed-9e7fba689d10.gif)

**Comment by qjebbs** [comment]
Please update to latest, and scroll preview to bottom, watch the snap indicator.

**Comment by Zooce** [comment]
This is working pretty nicely, thanks!

A couple things:
1) This doesn't seem to be working in the Insiders build - maybe something to check out
2) At first I thought it was a button I needed to click, but after playing around with it, and re-reading the extension's details, I realized it wasn't a button - just wanted to give that feedback now in case someone else says something later

BTW - thanks for getting this done so quickly!

**Comment by qjebbs** [comment]
Scroll and snap is more smooth, we need no buttons, but indicators to tell what is happening. Maybe we can consider how to improve the indicator styles.

**Comment by qjebbs** [comment]
I tend to hide all indicators by default in next release, since they brings misunderstanding and distracting. Snap will work just like what is is now.
